README.md

```diff
@@ -1,162 +1,168 @@
 # awesome-icon
-A curated list of awesome frameworks, libraries, software, and other resources for the Icon blockchain ecosystem
-
+A curated list of awesome frameworks, libraries, software, and other resources for the ICON blockchain ecosystem
+ 
 _Note that smart contracts decentralized applications also function as libraries and software services with publicly accessible APIs. See the [composability](https://docs.icon.community/icon-stack/smart-contracts/composability) and [remote-procedure-call](https://docs.icon.community/icon-stack/client-apis/json-rpc-api) topics on the official documentations to learn more_
-
+ 
 # Contents
-
+ 
 - [Projects](#projects)
-  - [Core Icon blockchain](#core-icon-blockchain)
-  - [Decentralized finance](#decentralized-finance)
-  - [Exchanges](#exchanges)
-  - [Execution environment](#execution-environment)
-  - [Games](#games)
-  - [Governance](#governance)
-  - [Interoperability](#interoperability)
-  - [NFT](#nft)
-  - [Oracle](#oracle)
-  - [RPC](#rpc)
-  - [Templating](#templating)
-  - [Testing](#testing)
-  - [Tracker](#tracker)
-  - [Wallet](#wallet)
+ - [Core ICON blockchain](#core-icon-blockchain)
```

Comment:
Keep the indentation here. Do a preview of the file to see what I mean. I want the project subsections to be indented below the Project section